packages/react-graphql/src/hooks/query.ts

```diff
@@ -119,17 +119,42 @@ export default function useQuery<
       return;
     }
 
+    let subscription: ZenObservable.Subscription | undefined;
```

Comment:
Belated review - It threw me that we don't import from zen-observable anywhere. I think we could lean on a bit of type inference to avoid having the direct dependency on zen-observable in the package.json:

```
let subscription: ReturnType<typeof queryObservable['subscribe']> | undefined;
```